api-description/forms/pk/get.json

```diff
@@ -5,55 +5,66 @@
         "status": 200,
         "response_data": {
             "name": "How have you been using the site?",
-            "url": "http://localhost:8000/forms/1",
+            "description": "Some additional text expanding on the above",
             "fieldsets": [
                 {
+                    "pk": 1,
                     "name": "Free text field",
                     "description": "Some additional text expanding on the above",
                     "fields": [
                         {
-                            "id": 1,
+                            "pk": 1,
                             "name": "How did you discover the site?",
                             "help_text": "Search engine, friend...",
-                            "field_type": "free_text"
+                            "field_type": "free_text",
+                            "required": true,
+                            "answers": []
                         }
                     ]
                 },
                 {
+                    "pk": 2,
                     "name": "Numeric fields",
                     "description": "Some additional text expanding on the above",
                     "fields": [
                         {
-                            "id": 2,
+                            "pk": 2,
                             "name": "What is answer to the ultimate question of life, the universe, and everything?",
                             "help_text": null,
-                            "field_type": "number"
+                            "field_type": "number",
+                            "required": false,
+                            "answers": []
                         },
                         {
-                            "id": 3,
+                            "pk": 3,
                             "name": "How much percentage does it take?",
                             "help_text": null,
-                            "field_type": "percentage"
+                            "field_type": "percentage",
+                            "required": true,
+                            "answers": []
                         }
                     ]
                 },
                 {
+                    "pk": 3,
                     "name": "Choice fields",
                     "description": "Some additional text expanding on the above",
                     "fields": [
                         {
-                            "id": 4,
+                            "pk": 4,
```

Comment:
I'll make it `id` then.  IIRC Django models have both, they're just aliases of one another by default.